### DISCUSSION TITLE
Who's using uniforms?

### DISCUSSION THREAD

**Discussion by radekmie** [vazco/uniforms#886] [Show and tell]
How about **_Who's using uniforms?_** section? As the new website is here, we'd like to _finally_ make it live! If you want in, post your project name, logo, and, optionally, let us know, _how_ you are using uniforms. Later we'll simply accept PRs.

**Comment by macrozone** [comment]
cool idea, 

we can also show some examples how you can integrated it with these packages.

E.g. with my manul-admin, you can create a simple content-management-solution for your app (my goal was to create a react-version of https://github.com/orionjs/orion).

Maybe you start the section and i'll add my examples in a PR.

Until then: happy new year, @radekmie !

**Comment by radekmie** [comment]
@macrozone It's nice to hear that! I've analysed [panter/manul-admin](https://github.com/panter/manul-admin) and I have to admit two things: I've never done anything in the _mantra-style_ and it looks great - really extendable and easy to start with. More examples would be surely useful, but it's already interesting to use!

Also, I'll create this section soon.

**Comment by radekmie** [comment]
@macrozone Can I start with your projects in this section?

**Comment by radekmie** [comment]
We can also make an `examples/` directory, with integration examples, like file inputs - set of `.md` files with a description, usage details and the source of course. What do you think?

**Comment by radekmie** [comment]
@macrozone

**Comment by cyclops24** [comment]
@radekmie I'm working on a private project now and can't post link here, but I have some idea based on this awesome tool. I post it here later man. 😉 
Your are my hero, I love uniforms. Again thanks for it. 👍

**Comment by radekmie** [comment]
Reactivation!

**Comment by wtrocki** [comment]
AeroGear community is looking to use it more with react apps and templates.

https://AeroGear.org
https://Graphback.dev

**Comment by tombacski** [comment]
Hi Guys I'm Tom from Vazco. We are creating new uniforms site and would like to know who is using the uniforms and how. 

If your company is using uniforms, please let us know who you are. We'll be grateful  👍 

Ideally:
- share a link to the project 👍 
- (optionally) let us know how you're using uniforms 🥇 

It will help us also to focus on the right things while developing the package in the future.

Thx!

**Comment by ArchiCroc** [comment]
I'm using Uniforms with the internal enterprise apps I write.  I liked that it's flexible with what UI framework we used and was integrated with simple-schema that we were already using.  I've published an example of my work using Uniforms at https://github.com/ArchiCroc/pup/tree/feature/combined .

**Comment by macrozone** [comment]
https://github.com/react-page/react-page/tree/master/packages/plugins/createPluginMaterialUi :tada:

will integrate it more, that is just a first step ;-)

**Comment by kyleder** [comment]
We're using Uniforms internally for complex patient charting at Boulder Care (https://boulder.care).

**Comment by vfonic** [comment]
Here at CivicLift, we use uniforms for all our forms.

Here's the link to the website: https://www.civiclift.com/Litchfield-Hills-CT

**Comment by tombacski** [comment]
@vfonic Thanks for the message about uniforms and your project!

**Comment by tombacski** [comment]
We are happy to meet people who use uniforms in their projects. We still collect this kind of information about which projects use the package! So if you are using a package and you are happy  - let us know! :)

**Comment by znat** [comment]
We use uniforms in many forms of [Botfront](https://github.com/botfront/botfront), our open source chatbot platform

**Comment by harryadel** [comment]
We're big proponents of uniforms at [Cult of Coders](https://www.cultofcoders.com/), especially as fellow Meteor developers. :smile:

**Comment by kletkeman** [comment]
I like uniforms quite a lot for a Meteor project for a customer. I use the materialize component in Meteor with uniforms-unstyled and add my own styling. With very little code I have a very nice way of showing subforms indented with a light dashed span tying it together, and I use an inlinearray tag (layout="subform|inlinearray" in uniforms option) to display tags in a wrapping array with custom buttons for add and remove, and to display a select field inline as radio buttons, or a multi-select field by adding "checkboxes" to the uniforms options. All in all, it is proving quite flexible. I am also able to incorporate tiles in a form with reorder using FlipMove etc. That way, I can add edit buttons to launch uniforms modally so that I end up with two levels of forms. Very useful, and thanks for this component.

**Comment by anasvn** [comment]
I have been looking this library since two days. Interesting. I did not used it yet. But still checking possibilities. This library remind me an angular package **[formly](https://github.com/ngx-formly/ngx-formly)**. Which very useful for form creation. I hope this package has similar features as **[formly](https://github.com/ngx-formly/ngx-formly)**.

**Comment by mvgijssel** [comment]
We're starting to use Uniforms at [HackerOne](https://www.hackerone.com/)! We've created components for our Design System and copied the playground components to our public storybook at https://storybook.inverselink.com/?path=/story/playground-form-builder--default. Uniforms really enables our developers to write frontend forms a lot faster and (maybe more importantly) in a consistent way ❤️ !
